### ISSUE TITLE
Enhance geospatial data handling

### ISSUE THREAD

**Issue by teklal** [IFRCGo/go-frontend#938]
This is a master ticket around improved collection, editing and rendering of geospatial data.

Related tickets:

**NOTE: This list was updated 7 October 2020 to reflect whether closed or moved to new issue.**

- Ref #614 - moved to #1642 
- Ref #619 - moved to #1643
- ~~Ref #784~~ - CLOSED
- ~~Ref #806~~ - CLOSED
- ~~Ref #850~~ - CLOSED
- ~~Ref #897~~ - CLOSED
- Ref #903 - moved to #1642 
- Ref #904 - moved to #1642 
- ~~Ref #936~~ - CLOSED
- ~~Ref #940~~ (Not totally closed but all geo stuff done)
- Ref #976 - moved to #1643
- Ref #1126 - moved to #1642 
- ~~Ref #1162~~ - CLOSED
- Ref #1188 - moved to #1643
- ~~Ref #1276~~ - CLOSED
- ~~Ref https://github.com/IFRCGo/go-api/issues/470~~ - CLOSED
- ~~Ref https://github.com/IFRCGo/go-frontend/issues/1422~~ - CLOSED
- Ref https://github.com/IFRCGo/go-frontend/issues/1445 - moved to #1643
- Ref https://github.com/IFRCGo/go-frontend/issues/1446 - moved to #1643
- Ref https://github.com/IFRCGo/go-frontend/issues/1447 - moved to #1643
- Ref https://github.com/IFRCGo/go-frontend/issues/1456 - moved to #1643

@batpad reflecting discussions around Mapbox and "canonical source of truth" of IFRC geospatial data.

**Comment by nanometrenat** [comment]
We have again had another instance in Production where a field report cannot be submitted because the countries dropdown and the database did not line up.  xref #936

**Comment by nanometrenat** [comment]
Known country config issues (not definitive, just adding as we spot them):

- Regional pages - Macao is showing as a country (AP)
- Field report country dropdown - Aland Island, Puerto Rico, New Caledonia all showing as countries

**Comment by nanometrenat** [comment]
Puerto Rico also currently getting GDACS emergencies auto-created:  https://go.ifrc.org/emergencies/4236

**Comment by LukeCaley** [comment]
Noting here that we should review the [MapAction data naming convention](https://guides.mapaction.org/gis_im_data_naming.html) as inspiration for geospatial data management.  

Note also the [MapAction GitHub](https://github.com/mapaction/data-name-lookup-tables/tree/master/200_data_name_lookup) is the single source of truth and used for the 'Moonshot' reference map automation project. 

cc @geohacker @batpad

**Comment by LukeCaley** [comment]
Adding for reference a link to OCHA's [Common Operational Datasets dashboard ](https://data.humdata.org/dashboards/cod)(and associated links) cc @geohacker

**Comment by jhenshall** [comment]
Latest geo update is that we are currently prioritising for the geo api integration at next release as outlined in #1534. It includes some issues already referred to on this ticket.

**Comment by jhenshall** [comment]
@batpad @geohacker - just for visibility, I am moving this out of 4.3 as it is difficult to UAT a ticket of other tickets. Once this release is done, would be great to review all remaining geo issues and close this one (which has been a great single-source ticket) but with a preference for more defined tickets. Cheers.

**Comment by jhenshall** [comment]
This ticket has been super helpful thanks to @teklal for setting up and @nanometrenat for adding to it. With a mix of closed and open issues for various tasks, I have updated the initial list marking which are closed and moved outstanding issues to #1642 for mapbox styling and functionality or #1643 for geo data. This legendary ticket will now be closed. Thank you! 🙏
